spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -11,9 +11,9 @@ class { 'foreman::cli':
       password    => 'changeme',
     }
 
-    if $facts['osfamily'] == 'RedHat' {
-      include foreman::cli::ansible
-      include foreman::cli::azure
+    if $facts['os']['family'] == 'RedHat' {
+      include ::foreman::cli::ansible
+      include ::foreman::cli::azure
```

Comment:
This is the old style we got rid of
```suggestion
      include foreman::cli::ansible
      include foreman::cli::azure
```